src/routes/bitbucket/proxy/index.ts

```diff
@@ -49,7 +49,7 @@ export function proxyRoutes(runner: Runner, client: BitbucketClient) {
           errors.push(...landChecks.errors);
           if (landChecks.existingRequest) {
             existingRequest = true;
-            errors.push('This PR has already been queued, patience young padawan');
+            errors.push('Pull request has already been queued');
```

Comment:
Getting serious now are we

Reply:
🫡 Too serious? I can only handle so many star wars references in my life